scripts/lint.sh

```diff
@@ -7,50 +7,28 @@ if [[ $(flutter format -n .) ]]; then
 fi
 
 flutter pub get
-result=$(dartanalyzer lib/)
-if ! echo "$result" | grep -q "No issues found!"; then
-  echo "$result"
-  echo "dartanalyzer issue: lib"
-  exit 1
-fi
+
+# We need to run pubget on all the examples
+for f in doc/examples/**/pubspec.yaml; do
+  d=$(dirname $f)
+  cd $d
+  flutter pub get
+  cd - > /dev/null
```

Comment:
discard cd log? what does cd log? haha and can it go wrong?

Reply:
I don't know to be honest, I just kept the same command that we had previously